Makefile

```diff
@@ -1,36 +1,14 @@
-# HURUmap Makefile
-# ----------------
-
-# HURUmap base
-
-base:
-	docker build --no-cache -t codeforafrica/hurumap-base:0.1.2 contrib/base
-	docker build -t codeforafrica/hurumap-base:latest contrib/base
-	docker push codeforafrica/hurumap-base:0.1.2
-	docker push codeforafrica/hurumap-base:latest
-
-
-# HURUmap app
+COMPOSE = docker-compose
+DEVDOCKER = $(COMPOSE) exec web
```

Comment:
Following the "standard" for aleph incase anyone builds with that too - https://github.com/alephdata/aleph/blob/master/Makefile#L2

It's not perfect but I'm considering better to implement across more projects instead of setting our own standards for easy move between projects for our devs.